config/manager/overlays/debug/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+patchesStrategicMerge:
+- dlv_container.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../../base
```

Comment:
this structure is a bit confusing for me because there is circular inheritance happening between `base` and `manager` folders: 
![original](https://user-images.githubusercontent.com/10904967/220678450-05e4f835-7fe2-4a35-ada5-85e0934855d5.png) 

Also I think the overlays folder should be in `config` folder and be sibling to config/base, which is more typical to see imo : 

![image](https://user-images.githubusercontent.com/10904967/220678848-d1c7132d-e209-4020-9721-ba3055fa3e5f.png)

Reply:
in case my arrows are confusing, the overlays/debug/kustomization looks like: 

```yaml
patchesStrategicMerge:
- dlv_container.yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- ../../base   # inheriting from config/base
images:
- name: controller
  newName: quay.io/opendatahub/data-science-pipelines-operator
  newTag: latest

```

and `config/base/kustomization.yaml` looks like: 

```yaml
namespace: data-science-pipelines-operator
namePrefix: data-science-pipelines-operator-
resources:  # bundles all manifests not in overlays
- ../crd
- ../rbac
- ../manager

images:
- name: controller
  newName: quay.io/opendatahub/data-science-pipelines-operator
  newTag: latest
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
```